release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -118,18 +118,12 @@ detected.
 When CRI-O use is enabled, it is installed alongside `docker`, which currently
 is required to perform build and push operations to the registry. Over time,
 temporary `docker` builds can accumulate on nodes. You can optionally set the
-following to enable garbage collection, which adds a daemonset to clean out the
-builds:
+following to enable garbage collection. You should make sure to configure the
```

Comment:
I'd say:

following parameter to enable garbage collection. You must configure the node selector so that the garbage collector runs only on nodes where \`docker\` is configured for \`overlay2\` storage.

(in both files, please)

Reply:
ok, updated them.